bot_reddit.py

```diff
@@ -0,0 +1,21 @@
+import praw
+import creds
+
+r = praw.Reddit(
+    client_id=creds.client_id,
+    client_secret=creds.client_secret,
+    password=creds.password,
+    user_agent=creds.user_agent,
+    username=creds.username,
+)
+
+subreddit = r.subreddit(creds.subreddit)
+
+
+def send_message(elements):
+    for element in elements:
+        text = ">"+element["subtitle"] + "\n\n\n" \
```

Comment:
This could be cleaned up to be much more readable by using a multiline format string.
```python
REDDIT_MSG_FORMAT = """
>{subtitle}


* ⭐️: Rating  {stars}/5 ({tot_rating})

* 👥:  {students}

[Get Course]({url})
"""
```
Followed be its usage:
```python
REDDIT_MSG_FORMAT.format(**element)
````